tests/components/threshold/test_binary_sensor.py

```diff
@@ -16,7 +16,24 @@
 from tests.common import MockConfigEntry
 
 
-async def test_sensor_upper(hass: HomeAssistant) -> None:
+@pytest.mark.parametrize(
+    ("from_val", "to_val", "expected_position", "expected_state"),
+    [
+        (None, 15, "below", "off"),  # at threshold
```

Comment:
is to_val None a valid test case or is that covered by "cat"?

Reply:
I'll add those cases.